Feature/my

Karol-Plocharski opened the pull request.

Karol-Plocharski committed:
my commit

Karol-Plocharski committed:
Merge remote-tracking branch 'origin/master' into feature/my

Karol-Plocharski committed:
tworzenie użytkownika z name, surname, occupation, date

RafalRomanQW commented on src/dao/DepartmentDao.java:
```diff
@@ -1,4 +1,45 @@
+
 public class DepartmentDao {
-}
+    public DepartmentDao (String shortname, String fullname) {
```
Constructor does nohing

RafalRomanQW commented on src/dao/EmployeeDao.java:
```diff
@@ -1,4 +1,59 @@
+                System.out.println("Date of arrive " + " dd/MM/yyyy");
+
+                //DateFormat formatter = new SimpleDateFormat("EEEE dd MMM yyyy");
+                Date date = new SimpleDateFormat("dd/MM/yyyy").parse(input.nextLine());
```
clear unused comments

RafalRomanQW commented on src/dao/DepartmentDao.java:
```diff
@@ -1,4 +1,45 @@
+        for (DepartmentDto department : departments) {
+            System.out.print(department.getShortname());
+            System.out.print(department.getFullname());
+
```
delete unused enters

RafalRomanQW commented on src/utils/App.java:
```diff
@@ -1,15 +1,14 @@
 
 public class App {
-    public void run() {
+    public void run() throws ParseException {
```
Exception should be in try and catch

Karol-Plocharski committed:
Dostosowanie klasy DepartmentDao do wymagań + modyfikacja klasy DepartmentDto + zmiany w klasie App w celu uruchomienia aplikacji.
Uwagi: Potrzebne dostosowanie klasy DepartmentDao by wyświetlała poprawnie wpisane informacje. Stworzenie odpowiednich pętli w klasie App oraz stworzeniu modułu wyszukiwania.
